src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
It would be great to make a variable for it

Reply:
Fix this recommendation please